Dockerfile

```diff
@@ -6,14 +6,20 @@ RUN apt-get update && \
     clang \
```

Comment:
Consider indenting these arguments to `apt-get`.

Reply:
OK.